src/modules/create-market/components/create-market-form-review/create-market-form-review.jsx

```diff
@@ -53,14 +53,17 @@ export default class CreateMarketReview extends Component {
   }
 
   calculateMarketCreationCosts() {
-    augur.createMarket.getMarketCreationCostBreakdown({ universe: this.props.universe.id, meta: this.props.meta }, (err, marketCreationCostBreakdown) => {
-      if (err) return console.error(err)
-      // TODO add designatedReportNoShowReputationBond to state / display
-      this.setState({
-        gasCost: formatEtherEstimate(0), // FIXME real gas cost lookup
-        designatedReportNoShowReputationBond: formatEtherEstimate(marketCreationCostBreakdown.designatedReportNoShowReputationBond),
-        creationFee: formatEtherEstimate(marketCreationCostBreakdown.targetReporterGasCosts),
-        validityBond: formatEtherEstimate(marketCreationCostBreakdown.validityBond),
+    this.props.estimateSubmitNewMarket(this.props.newMarket, (err, gasCost) => {
+      if (err) console.error(err)
```

Comment:
Should this be `if (err) return console.error(err)`?

Larger question: should there be something besides console.error (for example, an error display component) so that the user is made aware that something has gone wrong?

Reply:
that was a question, we default to 0 as gas costs if this fails. I didn't see anything design considerations if estimation fails.

Reply:
do we want a notification? I'll add comment to ch task

Reply:
looking closer, I don't want to return out of the method just because estimate gas fails, still want to call `augur.createMarket.getMarketCreationCostBreakdown`, the user can still submit create market even though gas estimate call fails.